react-web/src/pages/certification/components/FileCoverageContainer.tsx

```diff
@@ -1,9 +1,27 @@
-import React from "react";
+import React, { useState } from "react";
+import Modal from "components/Modal/Modal";
+import parse from 'html-react-parser';
 
 const FileCoverageContainer: React.FC<{
     result: { [x: string]: any };
     githubLink: string;
-}> = ({ result, githubLink }) => {
+    coverageFile?: string
+}> = ({ result, githubLink, coverageFile = '' }) => {
+    const [isOpen, setIsOpen] = useState("")
+    const openModal = (file: string) => {
+        setIsOpen(file)
+    }
```

Comment:
`openModal == setIsOpen` just with the difference openModal returns `undefined`

if it's just for naming purpose:
```suggestion
    const openModal = setIsOpen
```

Reply:
Because, multiple Modal instances may need to be opened here based on the number of coverage files, and to differentiate the isOpen

Reply:
ok, but still, wrapping `setIsOpen` into another function, and calling it with the same parameter doesn't make any difference, am I right?

are these two snippets different ( besides the returning value )? 

```
const openModal = (file: string) => {
        setIsOpen(file)
    }
```
and 

```
const openModal = setIsOpen
```